index.js

```diff
@@ -23,6 +25,10 @@ exports.getAllCards = function () {
     return cards.getAll();
 }
 
+exports.getScriptFromDeck = function (deck) {
```

Comment:
If you're exporting the private function, why not make it public from the beginning?